lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -617,6 +619,8 @@ import qualified Cardano.Wallet.Primitive.Types.Tx.TxOut as TxOut
 import qualified Cardano.Wallet.Primitive.Types.UTxO as UTxO
 import qualified Cardano.Wallet.Primitive.Types.UTxOStatistics as UTxOStatistics
 import qualified Cardano.Wallet.Read as Read
+import Cardano.Wallet.Shelley.Transaction
+    ( calculateMinimumFee )
```

Comment:
wrong section

Reply:
@Unisay this import needs to move to the section above 🆙